sktime/forecasting/compose/tests/test_pipeline.py

```diff
@@ -478,6 +478,50 @@ def test_forecastx_logic():
     assert np.allclose(y_pred, y_pred_manual)
 
 
+@pytest.mark.skipif(
+    not _check_soft_dependencies("statsmodels", severity="none"),
```

Comment:
Where is `statsmodels` required? If I'm not wrong, `ARIMA` comes from `pmdarima`.

Reply:
yes - `_check_estimator_dep` is safer as it retrieves the correct dependencies.

Reply:
I changed it